web-console/src/lib/components/streaming/management/PipelineTable.tsx

```diff
@@ -312,10 +313,10 @@ const DetailPanelContent = (props: { row: Pipeline }) => {
           </Card>
         </Grid>
         <Grid item xs={5}>
-          <AnalyticsPipelineTput metrics={metrics.global} />
+          <AnalyticsPipelineTput metrics={metrics} keepMs={30000} />
```

Comment:
can this value come from a global constant?
this way you don't risk to forget changing it in only one place